src/casp/services/api/data_manager/sql_templates/cell_analysis/get_cells_by_ids.sql.mako

```diff
@@ -0,0 +1,6 @@
+<%
```

Comment:
Let's rename `sql_templates/cell_analysis` to something more descriptive. What's the commonality between all the templates you have (and you intend to have in the future) in this directory?

Reply:
Also, could you put documentation at the top of sql.mako templates (e.g. a brief description what the query is supposed to do)?

Reply:
How about `sql_templates/cell_operations` or `sql_templates/cell_table_operations`?

Reply:
To be clear, my issue is with the choice of word `analysis`. To me, analysis means doing a lot more than summarizing some statistics, making temp tables, querying by ID, etc. Operations, on the other hand, seems more relevant.